src/components/Help/HelpStepper.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import {
+    Stepper,
+    Step,
+    StepButton,
+    Button,
+    Grid,
+} from '@material-ui/core';
+import {
+  BACK_BUTTON,
+  NEXT_BUTTON,
+} from 'components/Help/HelpConstants';
+import 'components/Help/Stepper.scss';
+
+const getStepContent = (instructions, stepIndex) => {
+  const imagePaths = Object.keys(instructions).map(
+    (key) => instructions[key].imagePath
+  );
+
+  const altTexts = Object.keys(instructions).map(
+    (key) => instructions[key].altText
+  );
+
+  return (
+    <div className='box'>
+      <img src={imagePaths[stepIndex]} alt={altTexts[stepIndex]} />
+    </div>
+  );
+};
+
```

Comment:
Use this in code/JSX directly.

Reply:
mapping imagePath and altText removed, new code does a direct pull from the object based on activeStep.